content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -88,6 +88,15 @@ has all the information needed to configure a load balancer or proxy server. Mos
 contains a list of rules matched against all incoming requests. Ingress resource only supports rules
 for directing HTTP(S) traffic.
 
+If the `ingressClassName` is omitted, a [default Ingress class](#default-ingress-class) 
+should be defined.
+
+There are some ingress controllers, that work without the definition of a 
+default `IngressClass`. For example, the Ingress-NGINX controller can be 
+configured with a [flag](https://kubernetes.github.io/ingress-nginx/#what-is-the-flag-watch-ingress-without-class) 
+`--watch-ingress-without-class`. It is recommended though, to specify the 
+default Ingress class as shown [below](#default-ingress-class).
+
```

Comment:
These sentences appear quite similar further down below, but I believe that redundancy is better than users not understanding what to do.